### PR TITLE
Use std::unique_ptr instead of UniquePtr in plasticdeformer.cpp

### PR DESCRIPTION
**Pull request by tomosu**
Replace UniquePtr with std::unique_ptr in plasticdeformer.cpp

### CODE REVIEW

**Comment by tomosu** [comment]
Jenkins

File: toonz/sources/tnzext/plasticdeformer.cpp [hunk 1]
```diff
@@ -70,54 +70,21 @@ struct LinearConstraint {
...
 };
 
 //-------------------------------------------------------------------------------------------
 
-typedef UniquePtr<tlin::SuperFactors, SuperFactors_free> SuperFactorsPtr;
-typedef UniquePtr<double, freer> DoublePtr;
-typedef UniquePtr<TPointD, deleter> TPointDPtr;
+typedef std::unique_ptr<tlin::SuperFactors, SuperFactors_free> SuperFactorsPtr;
+typedef std::unique_ptr<double[], freer> DoublePtr;
+typedef std::unique_ptr<TPointD[], deleter> TPointDPtr;
```
Comment (skitaoka) [inline toonz/sources/tnzext/plasticdeformer.cpp]: These are unnatural definitions.

**Comment by tomosu** [comment]
Thank you for your suaggestion.
Removed custom deleters from type definitions, and replaced "malloc" with "new" on re-initialization.

**Comment by skitaoka** [comment]
Jenkins, test this.

**Comment by kogaki** [comment]
LGTM

**Pull request merged**